desktop/apps/editorial_features/test/routes.coffee

```diff
@@ -85,6 +87,12 @@ describe 'Venice route', ->
     _.defer => _.defer =>
       @res.render.args[0][1].videoGuide.get('title').should.eql 'Video Guide'
 
+  it 'Fetches sub articles', ->
+    @req = { params: { slug: 'venice' } }
+    routes.venice(@req, @res, @next)
+    _.defer => _.defer => _.defer =>
```

Comment:
All these defers make me think there's probably a cleaner way to approach this -- for now this works but lets think later on how we can clean this up.